db/migrate/20230117161036_create_orders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.integer :quantity
+      t.string :ref_code
+      t.float :total_price
+      t.string :status
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
well done. it can be better to associate it with the user and food. so we will be able to see who performed the order and which food was ordered